src/main/java/ChatBot.java

```diff
@@ -0,0 +1,67 @@
+public class ChatBot {
+    private final String BOT_NAME = "Big Bob";
+    private final int MAXIMUM_NUMBER_OF_TASK = 100;
```

Comment:
Good job declaring constants (in the correct format) instead of having "magic numbers" or "magic strings" that could be potentially confusing to the reader